vector/src/main/java/im/vector/util/RoomUtils.java

```diff
@@ -82,8 +86,8 @@
     public static Comparator<Room> getRoomsDateComparator(final MXSession session, final boolean reverseOrder) {
```

Comment:
-> we should have one comparator for rooms and one for the public rooms

-> session.getDataHandler().getStore(aLeftObj.getRoomId()) might slow down the sort